zipline/algorithm.py

```diff
@@ -829,11 +912,26 @@ def schedule_function(self,
                           date_rule=None,
                           time_rule=None,
                           half_days=True):
+        """Schedules a function to be called with some timed rules.
```

Comment:
What do you think of "Schedules a function to be called according to some timed rules."?